pyproject.toml

```diff
@@ -400,7 +407,9 @@ ignore = [
   "W191", # tab-indentation (formatter conflict)
   "W291", # trailing-whitespace (handled by formatter)
 ]
-dummy-variable-rgx = "Z|mass_numb|request|event"
+dummy-variable-rgx = "^(Z|mass_numb|request|event|_.*)$"
+# Allow potentially ambiguous Unicode characters with valid uses in plasma science
+allowed-confusables = ["–", "α", "β", "γ", "ι", "ν", "ο", "ρ", "σ", "τ", "υ", "×", "′", "∼", "∪", "⊤", "∨", "ℂ", "ℕ", "ℚ", "ℝ", "ℤ", "ℑ", "ℒ", "ℓ", "ℳ", "ℜ", "ℱ", "ℎ", "ℰ", "𝒪", "ℐ"]
```

Comment:
We don't use all these characters, so I also included characters that we might use in equations in docstrings in the future.  I did it by enabling RUF001, RUF002, and RUF003; copying and pasting my Unicode cheatsheet for mathematical characters and cat emojis into a docstring; and running ruff to see what characters got caught.

I ended up raising https://github.com/astral-sh/ruff/issues/13330 since that would make it easier to configure this.